investment_tracking/urls.py

```diff
@@ -1,5 +1,7 @@
 from django.urls import path
+from .views import InvestmentTrackingSaveView
 
 urlpatterns = [
+    path("startup/<int:startup_id>/save", InvestmentTrackingSaveView.as_view(), name = "save-startup")
```

Comment:
Please be careful with the names. Is this really where startups are stored, if startups are not stored in this place, but something else is stored, then surely a different name is needed.
*name = "save-startup" -> "followed startups"

Reply:
got it